javaparser-core-testing/pom.xml

```diff
@@ -87,6 +87,10 @@
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+        </dependency>
```

Comment:
This was probably caused by my cleanup for the Java 11 compilation PR from a little while ago.